src/config/datadog.rs

```diff
@@ -28,14 +30,15 @@ impl Default for Options {
         Self {
             enabled: default_enabled(),
             api_key: None,
+            configuration: "".to_owned(),
             reporting_interval_secs: default_reporting_interval_secs(),
         }
     }
 }
 
 /// By default, the Datadog feature is enabled.
 const fn default_enabled() -> bool {
-    false
+    true
```

Comment:
Datadog Pipelines integration is now enabled automatically where the following conditions are true:

1. A `[datadog]` section is defined.
2. `datadog.configuration` (A UUID provided by Datadog Pipelines) is given.
3. `datadog.api_key` is either defined explicitly, or is provided by `DATADOG_API_KEY` or `DD_API_KEY` in the environment.
4. `datadog.enabled` is not explicitly `false`.